test/e2e/azure/account.go

```diff
@@ -0,0 +1,124 @@
+package azure
+
+import (
+	"encoding/json"
+	"log"
+	"os/exec"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Account holds the values needed to talk to the Azure API
+type Account struct {
```

Comment:
Could we just let `acs-engine` do all of this since we can now do `acs-engine deploy ...` that will do the login, generation and deployment?